models/user.js

```diff
@@ -3,27 +3,13 @@ const mongoose = require('mongoose'),
 
 //Schema Config
 var userSchema = new mongoose.Schema({
-   email: {
+    email: {
        type: String,
-     //required: true,
-     //unique: true,
+       required: true,
+       unique: true,
        lowercase: true,
-       trim: true,
-   },
-   username: {
-       type: String,
-     //required: true,
-     //unique: true,
-       minlength: 3,
-       maxlength: 12,
        trim: true
    },
-   password: {
-       type: String,
-       //required: true,
-       trim: true,
-       minlength: 5        
-   },
```

Comment:
Doesn't each user still need a username and password?

Reply:
I'm pretty sure passport-local-mongoose is handling the. username and password stuff.  I verified it works with the User schema not including username, password